specification/appconfiguration/data-plane/Microsoft.AppConfiguration/preview/2023-05-01-preview/appconfiguration.json

```diff
@@ -0,0 +1,2079 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "version": "2023-05-01-preview",
+    "title": "Azure App Configuration"
+  },
+  "schemes": [
+    "https"
+  ],
```

Comment:
You should specify a security scheme and security requirement for your operations. See the documentation
in the [Azure API Style Guide](https://github.com/Azure/azure-api-style-guide/blob/main/openapi-style-guide.md#security).

Reply:
I haven't modified this part of the spec in this PR. The only changes are under /kv and /revisions paths.

Reply:
@avanigupta 

Although it's not a change, if it's a requirement that was omitted in previous iterations we should try to resolve it in this one.

Reply:
Added securityDefinitions.